src/arguments.cpp

```diff
@@ -107,6 +107,7 @@ static const Multiplier UNIVERSAL[] = {{'n', 1}, {'u', 1000}, {'m', 1000000}, {'
 //     title=TITLE      - FlameGraph title
 //     minwidth=PCT     - FlameGraph minimum frame width in percent
 //     reverse          - generate stack-reversed FlameGraph / Call tree
+//     target-cpu=CPU   - sample threads on a specific CPU (`-e cpu` only, default: -1)
```

Comment:
Don't forget to update `ProfilerOptions.md` as well.

Reply:
a0934a229cbe60e472037c3e707fdc8b2cc6cb5d